QUICKSTART.md

```diff
@@ -16,39 +16,40 @@ We use the following data type, slightly simplified from [`Claim.daml`](./daml/C
 
 ```Haskell
 data Claim a
- = zero
- | one a
- | give (Claim a)
- | and with lhs: Claim a, rhs: Claim a
- | or with lhs: Claim a, rhs: Claim a
- | scale with k: Date -> Decimal, claim: Claim a
- | when with predicate: Date -> Bool, claim: Claim a
- | anytime with predicate : Date -> Bool, claim: Claim a
- | until with predicate : Date -> Bool, claim: Claim a
+ = Zero
```

Comment:
Not sure whether this was a merge conflict or you changed deliberately:

We want to encourage lower-case constructors for users.

The reason is that `And` is actually of the form `And c c [c]`, not what is shown below. And this precludes the use of infix like ```a `And` b```. However, the lower-case constructor supports this i.e. ```a `and` b```. I even considered only exporting the lower case constructors, but we need the others for pattern matching.

So can you please switch back to using lower-case constructors? Thanks!

Reply:
moved back to lowercase constructors